src/preact/base-element.js

```diff
@@ -726,6 +726,19 @@ function collectProps(Ctor, element, ref, defaultProps) {
       devAssert(def.attrs);
       value = def.parseAttrs(element);
     }
+    if (def.attrPrefix) {
+      const currObj = {};
+      const attrs = element.attributes;
+      for (let i = 0; i < attrs.length; i++) {
+        const attrib = attrs[i];
+        if (attrib.specified && attrib.name.startsWith(def.attrPrefix)) {
```

Comment:
What's the `specified` for? Is it still even a thing?

Reply:
I thought it might be necessary for browser compatibility but looks like it's not really necessary.